packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -35,7 +37,7 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, any> {
     let { isTooltipVisible } = this.state;
     return (
       <div
-        className='ms-TooltipHost'
+        className={ css('ms-TooltipHost', styles.host) }
```

Comment:
not gonna split this out for 1 rule for a subcomponent, let's get these in with minimal changes.